selfdrive/car/hyundai/fingerprints.py

```diff
@@ -1699,6 +1699,14 @@
       b'\xf1\x00GL3_ RDR -----      1.00 1.02 99110-L8000         ',
     ],
   },
+  CAR.KIA_K5_HEV_2024: {
+    (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\x00DL3HMFC AT KOR LHD 1.00 1.02 99210-L2500 230911'
```

Comment:
Since this version is bumped a few years from the last one (and it's CAN FD), we will need to check to make sure that the lateral accel factor is what we expect (checking now): https://github.com/commaai/openpilot/pull/30220/files#r1513777753